docs/config/README.md

```diff
@@ -61,17 +61,20 @@ This is the list of prompt-wide configuration options.
 
 ### Options
 
-| Variable       | Default                       | Description                                            |
-| -------------- | ----------------------------- | ------------------------------------------------------ |
-| `add_newline`  | `true`                        | Add a new line before the start of the prompt.         |
-| `prompt_order` | [link](#default-prompt-order) | Configure the order in which the prompt module occurs. |
-| `scan_timeout` | `30`                          | Timeout for starship to scan files (in milliseconds).  |
+| Variable                | Default                       | Description                                                  |
+| ----------------------- | ----------------------------- | ------------------------------------------------------------ |
+| `clear_before_printing` | `false`                       | Clear the terminal to the bottom before printing the prompt. |
+| `add_newline`           | `true`                        | Add a new line before the start of the prompt.               |
+| `prompt_order`          | [link](#default-prompt-order) | Configure the order in which the prompt module occurs.       |
+| `scan_timeout`          | `30`                          | Timeout for starship to scan files (in milliseconds).        |
 
 ### Example
 
 ```toml
 # ~/.config/starship.toml
 
+# Clear the terminal from the prompt to the bottom of the terminal before printing.
```

Comment:
It looks like changes from #739 might've gotten mixed into this PR.

Reply:
I apparently managed to fork off of the wrong branch 🙄. Last night was apparently not a good coding moment for me.